trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/get-back-link-href.ts

```diff
@@ -1,8 +1,8 @@
 // "Recommended Security and Performance Max: 2048 CHARACTERS" - https://stackoverflow.com/a/48230425
 const MAX_URL_CHARACTERS = 2048;
-const BASE_URL_CHARACTER_ALLOWANCE = 100;
+const BASE_URL_CHARACTER_ALLOWANCE = 150;
 
-export const getBackLinkUrl = (reportId: string, feeRecordIds: number[]): string => {
+export const getAddToExistingPaymentBackLinkToUtilisationReportReconciliationPage = (reportId: string, feeRecordIds: number[]): string => {
```

Comment:
I think perhaps this is bordering on a little too long... how about `getLinkToPremiumsPaymentsTab` and then perhaps rename `feeRecordIds` to `selectedFeeRecordIds` to make it clear why we are passing them in?